app/jobs/calendar_importer/event_resolver.rb

```diff
@@ -72,65 +72,27 @@ def determine_location_for_strategy
     data.partner_id = calendar.partner_id
   end
 
-  def event_strategy(place, address: nil)
+  def event_strategy(partner, address: nil)
     if data.has_location?
-      # place = 'attempt to match location'
-      # address = 'calendar.place.address || location'
-
-      # try to find the place
-      place = Partner.fuzzy_find_by_location(event_location_components)
-
-      # try to use that address
-      address = place&.address
-
-      # if no place, try to find an address
-      # (This only executes if there is no place given)
-      address ||= Address.search(data.location, event_location_components, data.postcode)
-
-      # if we have an address, try to use the place that address points to
-      # (Only runs if the fuzzy find and calendar.place are both nil)
-      place ||= address&.partners&.first
-
-      # NOTE: What happens if address has zero partners and the earlier assignments fail?
-      #       'place' is nil in this instance
-      # NOTE: Address.search can also return Nil if there are no event_location_components, or
-      #       if the address failed to save
-      # Both of these will cause the event to fail validation with "No place or address could be created/found (etc)"
-
-    elsif place.present?
+      partner = Partner.fuzzy_find_by_location(event_location_components)
```

Comment:
I might be missing something but it seems like this is only looking at the partner name, not postcode.

Reply:
argh yes. Have fixed!